canonical-web-and-design/vanilla-framework
# Rename "tab buttons" to "segmented control"

## Done

- Rename `scss` files from "tab-buttons" to "segmented-control" (but keep the old classes for compatibility)
- Update documentation
- Add example for the "new" component

Fixes https://github.com/canonical-web-and-design/vanilla-squad/issues/1328

## QA

- Open [demo](https://vanilla-framework-4410.demos.haus/)
- In the examples section, check that both "tab buttons" and "segmented control" work in its different variants.
- Review updated documentation:
  - Segmented control

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
